kedro-datasets/kedro_datasets/ibis/table_dataset.py

```diff
@@ -154,7 +154,8 @@ def hashable(value):
             import ibis
 
             config = deepcopy(self._connection_config)
-            backend = getattr(ibis, config.pop("backend"))
+            backend_attr = config.pop("backend") if config else None
```

Comment:
Shall we use a default value for the `backend`, or are we okay with raising an error if the key is missing?

Reply:
I'm not sure about this, @DimedS, thoughts?

Reply:
I think ibis have a default backend for duckdb. We should stick with the underlying libraries as much as we could. If ibis would fail without the backend argument we should probably do the same.

Do check the default as I only say  this from memory.